service/src/main/java/tds/config/repositories/impl/ConfigRepositoryImpl.java

```diff
@@ -0,0 +1,123 @@
+package tds.config.repositories.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.stereotype.Repository;
+import tds.config.ClientSystemFlag;
+import tds.config.ClientTestProperty;
+import tds.config.repositories.ConfigRepository;
+
+import javax.sql.DataSource;
+import java.util.List;
+
+@Repository
+public class ConfigRepositoryImpl implements ConfigRepository {
+    private static final Logger LOG = LoggerFactory.getLogger(ConfigRepositoryImpl.class);
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public ConfigRepositoryImpl(DataSource dataSource) {
+        this.jdbcTemplate = new NamedParameterJdbcTemplate(dataSource);
+    }
```

Comment:
I believe this service will only read so the separation of reads/writes isn't necessary.  Have multiple read replicas with different connection Strings is something we will need to implement, but I think whatever solution we decide will be common across all the services and can be done later.